outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "policy_arn" {
+  description = "ARN for the new policy"
+  value = "${aws_iam_policy.policy.arn}"
+}
+
+output "access_key_id" {
+  description = "Access key id for the credentials"
+  value = "${aws_iam_access_key.key.id}"
+}
+
+output "secret" {
```

Comment:
It'd be better to call this `secret_access_key`, just so we're using the same naming convention as AWS.

Reply:
this was renamed in the documentation, outputs, example docs and outputs